Assets/SEE/XR/KeyboardKey.cs

```diff
@@ -0,0 +1,111 @@
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+/// <summary>
+/// This class handles a single keyboard key.
+/// This script is based on this tutorial: https://www.youtube.com/watch?v=vTonHBr4t4g
+/// </summary>
+public class KeyboardKey : MonoBehaviour
+{
+    /// <summary>
+    /// The character of the key.
+    /// </summary>
+    public string character;
+    /// <summary>
+    /// The shifted character.
+    /// </summary>
+    public string shiftCharacter;
```

Comment:
Because we are setting the keyLabel.text (TMP_Text) to these, we would need to convert it again with .ToString, so I would say it is easier if we just let them be strings in the first place.

Reply:
I see, makes sense 👍🏻